console/console-init/ui/src/modules/iot-detail/components/IoTProjectDetailHeader/IoTProjectDetailHeader.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import { IoTProjectDetailHeader } from "./IoTProjectDetailHeader";
+
+describe("IoT Project header", () => {
+  test("it renders a iot project header", () => {
+    const props = {
+      projectName: "iot-project-name",
+      type: "iot-project-type",
+      status: "Active",
+      isEnabled: true,
+      changeEnable: () => {},
```

Comment:
you can jest.fn() for creating functions. 
  changeEnable: jest.fn(),
      onEdit:jest.fn(),
      onDelete: jest.fn()